guides/source/routing.md

```diff
@@ -1232,41 +1296,28 @@ Rails now creates routes to the `CategoriesController`.
 | PATCH/PUT | /kategorien/:id            | categories#update  | category_path(:id)      |
 | DELETE    | /kategorien/:id            | categories#destroy | category_path(:id)      |
 
-### Overriding the Singular Form
```

Comment:
I might have missed this elsewhere, but is there any reason why this was removed? I find it useful to have at least a tip or link mentioning custom inflections.

Reply:
Hm...I *think* we felt that inflections aren't to the Routing and maybe a bit extraneous. But open to adding back in.

Reply:
I guess a tip would be helpful: "if you need to override how routes are pluralized, check our inflector docs" or something along those lines.

Reply:
This can be a separate PR, not to block this one.

Reply:
Added it back (and reworded slightly). If you find it useful, someone else might as well :)